src/Flight/Flight.java

```diff
@@ -25,7 +26,7 @@ public Flight(){}
     // example: new Flight(1, "PHL", "BOS".....)
     // AllArgs constructor
     public Flight(int flightNumber, String originAirport, String destinationAirport,
-                  Date timeDeparture, Date timeArrival, short seatCount, int pilot, int airline){
+                  LocalDateTime timeDeparture, LocalDateTime timeArrival, short seatCount, int pilot, int airline){
```

Comment:
Was the reason for the data type switch from date to localdatetime to account for change in time zone?